convenience.js

```diff
@@ -176,3 +180,17 @@ const InjectionsHandler = new Lang.Class({
         object[name] = original;
     }
 });
+
+/**
+ * Return the actual position reverseing left and right in rtl
+ */
+function getPosition(settings) {
```

Comment:
Also not sure. But it is used by a few others files, so it must be in some shared third file.